### DISCUSSION TITLE
[Feedback] Install binary distribution

### DISCUSSION THREAD

**Discussion by Roavill2** [hyperledger/besu-docs#1255] [Feedback]
* Page: Install binary distribution
* URL: `/public-networks/get-started/install/binary-distribution/`

Not sure if I'm doing something wrong but following the doc I pulled down the latest .tar.gz release of Besu on Ubuntu 22.04, unzipped the tar with `tar -xvf`, changed into the unpacked directory but there is no `bin` directory.

Is there a command to generate this that isn't listed in the doc?

**Comment by gfukushima** [comment]
No extra commands needed, just make sure you've download the right file. It should contain a `bin` directory

**Comment by Roavill2** [comment]
Thanks for speed of response - realised I pulled down 23.1.0 which is in _Pre-release_

Pulled down 22.10.3 instead and that is working as expected. Thanks very much.
